docs/en-US/component/time-select.md

```diff
@@ -45,6 +47,7 @@ time-select/time-range
 | step                  | time step                                                | string             | —                       | 00:30       |
 | min-time              | minimum time, any time before this time will be disabled | string             | —                       | 00:00       |
 | max-time              | maximum time, any time after this time will be disabled  | string             | —                       | —           |
+| use-12-hour           | use 12-hour format                                       | boolean            | —                       | false       |
```

Comment:
I'd suggest here we use `clock-format` for this feature:

```ts
type ClockFormat = '12h' | '24h'

const clockFormat = {
  type: ColckFormat,
  default: '24h',
  acceptanceValue: ['12h', '24h']
}
```

Reply:
ok